pkg/dashboards/dashboard_test.go

```diff
@@ -28,10 +28,38 @@ import (
 
 func Test_dashboard_storage_methods(t *testing.T) {
 
+	//Create the defaultDB directory
+	defaultDBPath := "defaultDBs"
+	e := os.Mkdir(defaultDBPath, 0755)
+	if e != nil {
+		t.Fatalf("Failed to create defaultDB directory: %v", e)
+	}
```

Comment:
nit: use `err` instead of `e`, since we use `err` throughout the codebase as the error type value